pkg/storage/test/tuples.go

```diff
@@ -69,6 +69,62 @@ func ReadChangesTest(t *testing.T, datastore storage.OpenFGADatastore) {
 		})
 	})
 
+	t.Run("read_changes_with_start_time", func(t *testing.T) {
```

Comment:
Missing a test when both `start_time` and `continuationToken` is provided, it should give precedence to the token.

Reply:
Should we give precedence to the token or error out saying both cannot be provided? Erroring out would be more consistent but less flexible.

Reply:
I expect it could be somewhat common to append to existing query params:

```
> ReadChanges?start_time=2021-01-01T00:00:00.000Z
< { changes:[...], continuation_token: "123" }

> ReadChanges?start_time=2021-01-01T00:00:00.000Z&continuation_token=123
< { changes:[...], continuation_token: "456" }
```

maybe we just validate if `start_time < continuation_token`

Reply:
good point @elbuo8 I'll add one 👌

Reply:
> maybe we just validate if start_time < continuation_token

Keep in mind that we now support sorting, so depending on the sort direction the validation would change.

Reply:
@elbuo8 @jpadilla @poovamraj I guess this test is not super clear. It will actually do _paging_ to list all pages, meaning uses the start time first, then continuation token, so in a sense all lines are covered.

Let me think of a way to explicitly validate the results

Reply:
Resolved by dropping the requirement for `start_time < continuation_token` and suggesting that `continuation token takes precedence over start time`